src/pages/staking/index.js

```diff
@@ -1,422 +1,758 @@
+                  </li>
+                  <li>
+                    The{" "}
+                    <Link to="https://prater.launchpad.ethereum.org">
+                      Staking Launchpad
+                    </Link>{" "}
+                    will walk you through the process and hardware requirements
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "solo-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
+                    trackCustomEvent({
+                      eventCategory: `Staking Comparison`,
+                      eventAction: `Clicked`,
+                      eventName: "clicked solo staking",
+                    })
+                  }}
+                  to="/staking/solo/"
+                >
+                  More on solo staking
+                </StyledButtonLink>
+              </div>
+              <ColorH3 color="stakingGreen">Staking as a service</ColorH3>
+              <div
+                style={{
+                  gridArea: "saas-rewards",
+                  borderBottom: "1px solid #3335",
+                }}
+              >
+                <h4>Rewards</h4>
+                <ul>
+                  <li>
+                    Usually involves full protocol rewards minus monthly fee for
+                    node operations
+                  </li>
+                  <li>
+                    Dashboards often available to easily track your validator
+                    client
+                  </li>
+                </ul>
+              </div>
+              <div
+                style={{
+                  gridArea: "saas-risks",
+                  borderBottom: "1px solid #3335",
+                }}
+              >
+                <h4>Risks</h4>
                 <ul>
                   <li>
-                    <Translation id="page-staking-upgrades-li" />
+                    Same risks as solo staking plus counter-party risk of
+                    service provider
                   </li>
                   <li>
-                    <Translation id="page-staking-upgrades-li-2" />
+                    Use of your signing keys is entrusted to someone else who
+                    could behave maliciously
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "saas-reqs" }}>
+                <h4>Requirements</h4>
+                <ul>
+                  <li>Deposit 32 ETH and generate your keys with assistance</li>
+                  <li>Store your keys securely</li>
+                  <li>
+                    The rest is taken care of, though specific services will
+                    vary
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "saas-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
+                    trackCustomEvent({
```

Comment:
Let's remove these events - this is just a link, we can track this via pageviews.